### PR TITLE
add bed destroys team reward & win / lose team reward & indonesian language

### PR DESCRIPTION
**Pull request by josemarcellio**

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+
+public class Indonesia extends Language {
+
+    public Indonesia() {
+        super(BedWars.plugin, "id");
+
+        YamlConfiguration yml = getYml();
+        yml.options().copyDefaults(true);
+        yml.addDefault(Messages.PREFIX, "");
+        yml.addDefault("name", "Indonesia");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: this should be the name of Indonesian in your language. I'm waiting for a change.

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+                "&e&l Iron, Gold, Emerald, dan Diamond dari generator",
+                "&e&l untuk mengakses peningkatan agar lebih kuat.", "",
+                "&a▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬"));
+        yml.addDefault(Messages.ARENA_JOIN_DENIED_SELECTOR, "{prefix}&cMaaf, kamu tidak dapat bergabung dengan arena ini sekarang, gunakan klik kanan untuk menonton!");
+        yml.addDefault(Messages.ARENA_SPECTATE_DENIED_SELECTOR, "{prefix}&cMaaf, kamu tidak dapat menonton arena ini sekarang, gunakan klik kiri untuk bergabung!");
+        yml.addDefault(Messages.ARENA_JOIN_DENIED_NO_PROXY, "&cMaaf, kamu harus bergabung dengan arena menggunakan BedWarsProxy. \n&eJika kamu ingin menyiapkan arena, silahkan berikan anda akses bw.setup supaya bisa bergabung dengan server secara langsung!");
+        yml.addDefault(Messages.ARENA_SPECTATOR_TELEPORTER_GUI_NAME, "&8Teleporter");
+        yml.addDefault(Messages.ARENA_SPECTATOR_TELEPORTER_GUI_HEAD_NAME, "{prefix}{player}");
+        yml.addDefault(Messages.ARENA_SPECTATOR_TELEPORTER_GUI_HEAD_LORE, Arrays.asList("&7Health: &f{health}%", "&7Food: &f{food}", "", "&7Klik kiri untuk menonton"));
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: health and food are not translated :/

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.ARENA_SPECTATOR_LEAVE_ITEM_NAME, "&c&lKembali ke lobby");
+        yml.addDefault(Messages.ARENA_SPECTATOR_LEAVE_ITEM_LORE, Collections.singletonList("&7Klik kanan untuk pergi ke lobby!"));
+        yml.addDefault(Messages.ARENA_SPECTATOR_FIRST_PERSON_ENTER_TITLE, "&aSpectating &7{player}");
+        yml.addDefault(Messages.ARENA_SPECTATOR_FIRST_PERSON_ENTER_SUBTITLE, "&cSNEAK untuk keluar");
+        yml.addDefault(Messages.ARENA_SPECTATOR_FIRST_PERSON_LEAVE_TITLE, "&eExiting Spectator mode");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 4]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.GENERATOR_HOLOGRAM_TYPE_DIAMOND, "&b&lDiamond");
+        yml.addDefault(Messages.GENERATOR_HOLOGRAM_TYPE_EMERALD, "&a&lEmerald");
+        yml.addDefault(Messages.GENERATOR_HOLOGRAM_TIMER, "&eMuncul dalam &c{seconds} &edetik");
+        yml.addDefault(Messages.GENERATOR_UPGRADE_CHAT_ANNOUNCEMENT, "{prefix}{generatorType} Generator &etelah ditingkatkan ke tingkat &c{tier}");
+        yml.addDefault(Messages.FORMATTING_CHAT_LOBBY, "{level}{vPrefix}&7{player}{vSuffix}: {message}");
+        yml.addDefault(Messages.FORMATTING_CHAT_WAITING, "{level}{vPrefix}&7{player}{vSuffix}: {message}");
+        yml.addDefault(Messages.FORMATTING_CHAT_SHOUT, "{level}{vPrefix}&6[SHOUT] {team} &7{player}&f{vSuffix}: {message}");
+        yml.addDefault(Messages.FORMATTING_CHAT_TEAM, "{level}{vPrefix}&f{team}&7 {player}{vSuffix} {message}");
+        yml.addDefault(Messages.FORMATTING_CHAT_SPECTATOR, "{level}{vPrefix}&7[SPECTATOR] {player}{vSuffix}: {message}");
+        yml.addDefault(Messages.FORMATTING_SCOREBOARD_HEALTH, Arrays.asList("&c❤", "&aHealth"));
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 5]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.FORMATTING_ACTION_BAR_TRACKING, "&fTracking: {team} &f- Jarak: {distance}m");
+        yml.addDefault(Messages.FORMATTING_TEAM_WINNER_FORMAT, "      {TeamColor}{TeamName} &7- {members}");
+        yml.addDefault(Messages.FORMATTING_SOLO_WINNER_FORMAT, "                 {TeamColor}{TeamName} &7- {members}");
+        yml.addDefault(Messages.FORMATTING_GENERATOR_TIER1, "I");
+        yml.addDefault(Messages.FORMATTING_GENERATOR_TIER2, "II");
+        yml.addDefault(Messages.FORMATTING_GENERATOR_TIER3, "III");
+        yml.addDefault(Messages.FORMATTING_DESPAWNABLE_UTILITY_NPC_HEALTH, "▮ ");
+        yml.addDefault(Messages.FORMATTING_STATS_DATE_FORMAT, "yyyy/MM/dd HH:mm");
+        yml.addDefault(Messages.FORMAT_PAPI_PLAYER_TEAM_TEAM, "{TeamColor}[{TeamName}]");
+        yml.addDefault(Messages.FORMAT_PAPI_PLAYER_TEAM_SHOUT, "&6[SHOUT]");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 6]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.FORMAT_PAPI_PLAYER_TEAM_SPECTATOR, "&7[SPECTATOR]");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 7]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.MEANING_FULL, "Full");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 8]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.MEANING_SHOUT, "shout");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 9]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.MEANING_NOBODY, "Nobody");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 10]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.MEANING_NEVER, "Never");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 11]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.NEXT_EVENT_EMERALD_UPGRADE_II, "&fEmerald II");
+        yml.addDefault(Messages.NEXT_EVENT_EMERALD_UPGRADE_III, "&fEmerald III");
+        yml.addDefault(Messages.NEXT_EVENT_GAME_END, "&4Game End");
+        yml.addDefault(Messages.NEXT_EVENT_TITLE_ANNOUNCE_BEDS_DESTROYED, "&cTEMPAT TIDUR DIHANCURKAN!");
+        yml.addDefault(Messages.NEXT_EVENT_SUBTITLE_ANNOUNCE_BEDS_DESTROYED, "&fSemua tempat tidur telah dihancurkan!");
+        yml.addDefault(Messages.NEXT_EVENT_CHAT_ANNOUNCE_BEDS_DESTROYED, "&c&lSemua tempat tidur telah dihancurkan!");
+        yml.addDefault(Messages.NEXT_EVENT_TITLE_ANNOUNCE_SUDDEN_DEATH, "&cSudden Death");
+        yml.addDefault(Messages.NEXT_EVENT_SUBTITLE_ANNOUNCE_SUDDEN_DEATH, "");
+        yml.addDefault(Messages.NEXT_EVENT_CHAT_ANNOUNCE_SUDDEN_DEATH, "&cSUDDEN DEATH: &6&b{TeamDragons} {TeamColor}{TeamName} Naga!");
+        yml.addDefault(Messages.XP_REWARD_PER_MINUTE, "{prefix}&6+{xp} BedWars Experience Received (Play Time).");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 12]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.XP_REWARD_WIN, "{prefix}&6+{xp} BedWars Experience Received (Game Win).");
+        yml.addDefault(Messages.XP_REWARD_PER_TEAMMATE, "{prefix}&6+{xp} BedWars Experience Received (Team Support).");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 13]
```diff
@@ -0,0 +1,550 @@
...
+
+        yml.addDefault(Messages.MONEY_REWARD_PER_MINUTE, "{prefix}&6+{money} Coins (Play Time).");
+        yml.addDefault(Messages.MONEY_REWARD_WIN, "{prefix}&6+{money} Coins (Game Win).");
+        yml.addDefault(Messages.MONEY_REWARD_PER_TEAMMATE, "{prefix}&6+{money} Coins (Team Support).");
+        yml.addDefault(Messages.MONEY_REWARD_BED_DESTROYED, "{prefix}&6+{money} Coins (Bed Destroyed).");
+        yml.addDefault(Messages.MONEY_REWARD_FINAL_KILL, "{prefix}&6+{money} Coins (Final Kill).");
+        yml.addDefault(Messages.MONEY_REWARD_REGULAR_KILL, "{prefix}&6+{money} Coins (Regular Kill).");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 14]
```diff
@@ -0,0 +1,550 @@
...
+
+        /* Lobby Command Items */
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_NAME.replace("%path%", "stats"), "&eStats");
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_LORE.replace("%path%", "stats"), Collections.singletonList("&fRight-click to see your stats!"));
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_NAME.replace("%path%", "arena-selector"), "&eArena Selector");
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_LORE.replace("%path%", "arena-selector"), Collections.singletonList("&fRight-click to choose an arena!"));
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_NAME.replace("%path%", "leave"), "&eBack to Hub");
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_LORE.replace("%path%", "leave"), Collections.singletonList("&fRight-click to leave BedWars!"));
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 15]
```diff
@@ -0,0 +1,550 @@
...
+        /* Pre Game Command Items */
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_WAITING_ITEMS_NAME.replace("%path%", "stats"), "&eStats");
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_WAITING_ITEMS_LORE.replace("%path%", "stats"), Collections.singletonList("&fRight-click to see your stats!"));
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_WAITING_ITEMS_NAME.replace("%path%", "leave"), "&eBack to Lobby");
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_WAITING_ITEMS_LORE.replace("%path%", "leave"), Collections.singletonList("&fRight-click to leave the arena!"));
+        /* Spectator Command Items */
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 16]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_SPECTATOR_ITEMS_NAME.replace("%path%", "teleporter"), "&eTeleporter");
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_SPECTATOR_ITEMS_NAME.replace("%path%", "leave"), "&eBack to Lobby");
+        yml.addDefault(Messages.GENERAL_CONFIGURATION_SPECTATOR_ITEMS_LORE.replace("%path%", "leave"), Collections.singletonList("&fRight-click to leave the arena!"));
+
+        /* save default items messages for stats gui */
+        yml.addDefault(Messages.PLAYER_STATS_GUI_INV_NAME, "&8{player} Stats");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 17]
```diff
@@ -0,0 +1,550 @@
...
+        addDefaultStatsMsg(yml, "wins", "&6Wins", "&f{wins}");
+        addDefaultStatsMsg(yml, "losses", "&6Losses", "&f{losses}");
+        addDefaultStatsMsg(yml, "kills", "&6Kills", "&f{kills}");
+        addDefaultStatsMsg(yml, "deaths", "&6Deaths", "&f{deaths}");
+        addDefaultStatsMsg(yml, "final-kills", "&6Final Kills", "&f{finalKills}");
+        addDefaultStatsMsg(yml, "final-deaths", "&6Final Deaths", "&f{finalDeaths}");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 18]
```diff
@@ -0,0 +1,550 @@
...
+        addDefaultStatsMsg(yml, "beds-destroyed", "&6Beds Destroyed", "&f{bedsDestroyed}");
+        addDefaultStatsMsg(yml, "first-play", "&6First Play", "&f{firstPlay}");
+        addDefaultStatsMsg(yml, "last-play", "&6Last Play", "&f{lastPlay}");
+        addDefaultStatsMsg(yml, "games-played", "&6Games Played", "&f{gamesPlayed}");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 19]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault(Messages.SCOREBOARD_DEFAULT_STARTING, Arrays.asList("&f&lBED WARS", "&7{date} &8{server}", "", "&fMap: &a{map}", "", "&fPlayers: &a{on}/{max}", "", "&fStarting in &a{time}s", "", "§fMode: &a{group}", "&fVersion: &7{version}", "", "&e{server_ip}"));
+        yml.addDefault(Messages.SCOREBOARD_DEFAULT_PLAYING, Arrays.asList("&e&lBED WARS", "&7{date}", "", "&f{nextEvent} in &a{time}", "", "{TeamRedColor}R&f {TeamRedName}&f&f: {TeamRedStatus}",
+                "{TeamBlueColor}B&f {TeamBlueName}&f: {TeamBlueStatus}", "{TeamGreenColor}G&f {TeamGreenName}&f: {TeamGreenStatus}", "{TeamYellowColor}Y &f{TeamYellowName}&f: {TeamYellowStatus}",
+                "{TeamAquaColor}A &f{TeamAquaName}&f: {TeamAquaStatus}", "{TeamWhiteColor}W &f{TeamWhiteName}&f: {TeamWhiteStatus}", "{TeamPinkColor}P &f{TeamPinkName}&f: {TeamPinkStatus}",
+                "{TeamGrayColor}S &f{TeamGrayName}&f: {TeamGrayStatus}", "", "&e{server_ip}"));
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 20]
```diff
@@ -0,0 +1,550 @@
...
+
+        yml.addDefault("scoreboard.Doubles.playing", Arrays.asList("&e&lBED WARS", "&7{date}", "", "&f{nextEvent} in &a{time}", "", "{TeamRedColor}R&f {TeamRedName}&f&f: {TeamRedStatus}",
+                "{TeamBlueColor}B&f {TeamBlueName}&f: {TeamBlueStatus}", "{TeamGreenColor}G&f {TeamGreenName}&f: {TeamGreenStatus}", "{TeamYellowColor}Y &f{TeamYellowName}&f: {TeamYellowStatus}",
+                "{TeamAquaColor}A &f{TeamAquaName}&f: {TeamAquaStatus}", "{TeamWhiteColor}W &f{TeamWhiteName}&f: {TeamWhiteStatus}", "{TeamPinkColor}P &f{TeamPinkName}&f: {TeamPinkStatus}",
+                "{TeamGrayColor}S &f{TeamGrayName}&f: {TeamGrayStatus}", "", "&e{server_ip}"));
+
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 21]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault("scoreboard.3v3v3v3.playing", Arrays.asList("&e&lBED WARS", "&7{date}", "", "&f{nextEvent} in &a{time}", "", "{TeamRedColor}R&f {TeamRedName}&f&f: {TeamRedStatus}",
+                "{TeamBlueColor}B&f {TeamBlueName}&f: {TeamBlueStatus}", "{TeamGreenColor}G&f {TeamGreenName}&f: {TeamGreenStatus}", "{TeamYellowColor}Y &f{TeamYellowName}&f: {TeamYellowStatus}",
+                "", "&fKills: &a{kills}", "&fFinal Kills: &a{finalKills}", "&fBeds Broken: &a{beds}", "", "&e{server_ip}"));
+
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 22]
```diff
@@ -0,0 +1,550 @@
...
+        yml.addDefault("scoreboard.4v4v4v4.playing", Arrays.asList("&e&lBED WARS", "&7{date}", "", "&f{nextEvent} in &a{time}", "", "{TeamRedColor}R&f {TeamRedName}&f&f: {TeamRedStatus}",
+                "{TeamBlueColor}B&f {TeamBlueName}&f: {TeamBlueStatus}", "{TeamGreenColor}G&f {TeamGreenName}&f: {TeamGreenStatus}", "{TeamYellowColor}Y &f{TeamYellowName}&f: {TeamYellowStatus}",
+                "", "&fKills: &a{kills}", "&fFinal Kills: &a{finalKills}", "&fBeds Broken: &a{beds}", "", "&e{server_ip}"));
+
+        yml.addDefault(Messages.SCOREBOARD_LOBBY, Arrays.asList("&6&lBedWars,&4&lB&6edWars,&6&lB&4e&6dWars,&6&lBe&4d&6Wars,&6&lBed&4W&6ars,&6&lBedW&4a&6rs,&6&lBedWa&4r&6s,&6&lBedWar&4s,&6&lBedWars", "&fYour Level: {level}", "", "&fProgress: &a{currentXp}&7/&b{requiredXp}", "{progress}", "", "&7{player}", "", "&fCoins: &a{money}"
+                , "", "&fTotal Wins: &a{wins}", "&fTotal Kills: &a{kills}", "", "&e{server_ip}"));
+
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java [hunk 23]
```diff
@@ -0,0 +1,550 @@
...
+        //
+        yml.addDefault(Messages.SHOP_INDEX_NAME, "&8Quick Buy");
+        yml.addDefault(Messages.SHOP_INSUFFICIENT_MONEY, "{prefix}&cYou don't have enough {currency}! Need {amount} more!");
+        yml.addDefault(Messages.SHOP_NEW_PURCHASE, "{prefix}&aYou purchased &6{item}");
+        yml.addDefault(Messages.SHOP_ALREADY_BOUGHT, "{prefix}&cYou've already bought that!");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java]: needs translation

**Review by andrei1058 — CHANGES_REQUESTED** [review]
This is not fully translated.. I really appreciate, but you need to provide a full translation in order to have it merged...

**Comment by josemarcellio** [comment]
okay I will make fully translate

**Pull request closed without merging**